hbase-server/src/main/java/org/apache/hadoop/hbase/master/ActiveMasterManager.java

```diff
@@ -106,6 +114,31 @@ void handle(final String path) {
     }
   }
 
+  /*
+   * Fetches the active master's ServerName from zookeeper.
+   */
+  private void fetchAndSetActiveMasterServerName() {
+    LOG.debug("Attempting to fetch active master sn from zk");
+    try {
+      activeMasterServerName.set(MasterAddressTracker.getMasterAddress(watcher));
```

Comment:
Is it good to use `activeMasterServerName.getAndSet()` atomically? just wanted to understand how concurrently this might be updated
Thanks